app/templates/request-address-in-scotland.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base-' + display_region + '.html' %}
+
+{% block main %}
+
+    <h1 class="u-mt-l">{{ _('Your address is in Scotland') }}</h1>
+
+    <p>{{ address }}</p>
```

Comment:
Address not provided by RequestAddressInScotland class view when calling template so this is not displayed. Is it needed?

Reply:
Good spot - now removed